client/src/utils/track.ts

```diff
@@ -7,6 +7,14 @@ let configured = false;
 if (!configured) {
   Auth.configure(awsexports);
   Analytics.configure(awsexports);
+  // @ts-ignore
+  if (typeof AWSCShortbread !== "undefined") {
```

Comment:
Is there an additional check that we want to place here? Does it turn into a function? Does it need to be: 
```suggestion
  if (typeof AWSCShortbread !== undefined) {
```

Reply:
I tried that first and it doesn't work... actually it might be better to use `if (Build.isBrowser)` instead as is done in `track.ts` -- the code is there fully for testing

Reply:
Interesting why that wouldn't work 🤔